notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>Excellent question! Maybe the following interpretation can help:</p><p> </p><p>The vector field ∇_X Y is also called the covariant derivative of Y w.r.t. X.: i.e. how much Y changes when we move along X. (think about the traditional notion of derivative of a function: this is how much this function changes when we vary its input parameter slightly).</p><p> </p><p>If we think of ∇_X as a notion of derivative, then 2. is the equivalent of the leibniz rule i.e. (fg)' = f'g + fg' (because ∇_X f = X(f)). See bottom of page 10 and top of page 11 in the textbook. This involves the interpretation of tangent vectors as derivations, which is subtle but central to "differential" geometry.</p><p> </p><p>1. says that the rate of change of Y is scaled (by f) if we scale (by f) the vector along which we want to transport it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>ok, so essentially, the connection has to satisfy those conditions so that we can think of it is a "differential geometry" derivative?</p><p> </p><p>Maybe i will say the following:</p><p> </p><p>$\textbf{Now, let's consider the conditions that the connection must satsify.}$ Here, there is not much to dissect mathematically, but we can wonder why the connection must satisfy the conditions. The vector field $\nabla_{X} Y$ (remember $X$ and $Y$ are vector $\textit{fields}$ is also called the covariant derivative of $Y$ with respect to $X$ (i.e. how much $Y$ changes when we move along $X$. Note that this covariant derivative must also be a vector field because it contains information at each point.).&nbsp;</p><p> </p><p>If we think of $\nabla_{X}$ as a derivative of some sort, then it makes sense that $\nabla_{X}$ would have to follow some sort of rules. Condition 1 is the equivalent of saying $\frac{d(fX)}{dx}=f\frac{d(X)}{dx}$ in euclidean space. Similarly, condition 2 is the differential geometry equivalent of the product rule: $\frac{d(u*v)}{dx}= \frac{d(u)}{dx}*v +u*\frac{d(v)}{dx}.</p>